src/main/webapp/dashboard.js

```diff
@@ -1,49 +1,92 @@
+ * @typedef {Object} Mentee
+ * @property {String} image -the link to a mentee's image
+ * @property {String} name - the name of the mentee
+ * @property {String} starLink - the link to the action that rewards a given
+ *                               mentee with a star
+ */
+
 /**
  * A User
  * @typedef {Object} User
  * @property {String} bio - The bio of the user
- * @property {String} gitSrc - The user's github page 
+ * @property {String} gitSrc - The user's github page
  * @property {String} image - The User's profile picture
  * @property {String} location - the location of the user
  * @property {String} name - the name of the user
  * @property {String List} tags - The user's tags
  */
 
-// ELEMENTS_FOR_ABOUT_ME_SECTION
-/**
- * Elements from dashboard.html.
- * @type {HTMLElement}
- */
-const ABOUT_ME_CARD_DIV = document.getElementById('about-me-card-body');
-const USER_BIO = document.getElementById('user-bio');
-const USER_IMAGE = document.getElementById('user-image');
-const USER_NAME_AND_LOCATION = document.getElementById('user-name-location');
-const USER_TAG_ROW = document.getElementById('user-tag-row');
-const USER_GITHUB_BUTTON = document.getElementById('user-github');
-
 /**
  * Fake information for the hardcoded stage.
  */
+/** @const {String} */
 const dummyName = 'Dior';
 const dummyLocation = 'Sunnyvale CA.';
 const dummyBio = 'The cutest pug in The tech industry. Lorem ipsum dolor sit ' +
   'amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut' +
   ' labore et dolore magna aliqua.';
 const dummyImg = 'images/dior.jpg';
+const dummyGitSrc = '#';
+const dummyProjectName = 'Kubernetes';
+
+/** @const {String List} */
 const dummyTags = ['Weekly', '30 mins', 'Kubernetes', 'Open Sesame',
   'Documentation',
 ];
-const dummyGitSrc = '#';
 
-/**@const {User} */
+/** @const {User} */
 const dummyUser = {
   name: dummyName,
   location: dummyLocation,
   image: dummyImg,
   bio: dummyBio,
-  tags: dummyTags
+  tags: dummyTags,
+  gitSrc: dummyGitSrc,
 };
 
+/** @const {Mentee List} */
+const dummyMentees = [{
+    name: 'Richi',
+    starLink: '#givestar',
+    image: dummyImg
+  },
+  {
+    name: 'Obi',
+    starLink: '#givestar',
+    image: dummyImg
+  },
+  {
+    name: 'Sami',
+    starLink: '#givestar',
+    image: dummyImg
+  },
+];
+
+/** @const {Project} */
+const dummyProject = { name: dummyProjectName, mentees: dummyMentees };
+
+// ELEMENTS_FOR_ABOUT_ME_SECTION
+/**
+ * Elements from dashboard.html.
+ * @type {HTMLElement}
+ */
+const ABOUT_ME_CARD_DIV = document.getElementById('about-me-card-body');
+const USER_BIO = document.getElementById('user-bio');
+const USER_IMAGE = document.getElementById('user-image');
+const USER_NAME_AND_LOCATION = document.getElementById('user-name-location');
+const USER_TAG_ROW = document.getElementById('user-tag-row');
+const USER_GITHUB_BUTTON = document.getElementById('user-github');
+const AFFILIATIONS_DIV = document.getElementById('affiliations');
```

Comment:
I am wondering if these declarations warrant the all-uppercase constant case. The style guide states that "if any of that instance's observable state can change, it is almost certainly not a constant": https://google.github.io/styleguide/jsguide.html

Reply:
(curious if adding content to these elements counts as changing their observable states)

Reply:
Good catch, You're right!